examples/reference-entity-model/input/xmlCustomersWithRelatedEntities/customer1WithRelatedEntities.xml

```diff
@@ -0,0 +1,79 @@
+<Customer xmlns="customerInfo">
+  <CustomerID>401</CustomerID>
+  <Name type="preferred">
+    <FirstName>Jane</FirstName>
+    <LastName>Seymour</LastName>
+  </Name>
+  <Nicknames>
+    <nickname>Janie</nickname>
+    <nickname>Janey</nickname>
+  </Nicknames>
+  <Address>
+    <Shipping>
+      <Street>Covington Road</Street>
+      <City>Los Altos</City>
+      <State>California</State>
+      <Postal>94024-0241</Postal>
+    </Shipping>
+    <Billing>
+      <Street>Latham St</Street>
+      <City>Mountain view</City>
+      <State>California</State>
+      <Postal>94040-6783</Postal>
+    </Billing>
+  </Address>
+  <Orders>
+    <Order>
+      <OrderID>3002</OrderID>
+      <DateAndTime>2019/05/24-07:45:22</DateAndTime>
+      <FirstName>John</FirstName>
+      <LastName>Lithgow</LastName>
+      <SendToID>403</SendToID>
+      <Products>
+        <Product>
+          <Quantity>1</Quantity>
+          <ProductId>70</ProductId>
+          <Name>Skechers GOrun Razor +</Name>
```

Comment:
For manual testing, it'll be easier to just name these e.g. "Product 70" and "Product 90". It'll be good to test longer names to see how they show up in the UI, but that's a separate concern.